cmd/internal/find/primary/size.go

```diff
@@ -46,3 +43,34 @@ var sizePrimary = Parser.add(&primary{
 		return p, tokens[1:], nil
 	},
 })
+
+const sizeDetailedDescription = `
+-size [+|-]n[ckMGTP]
+
+Returns true if the entry's size attribute is n 512-byte blocks,
+rounded up to the nearest block. If n is suffixed with a unit,
+then the size is compared as-is to n scaled as:
```

Comment:
Not sure what `as-is` adds to this sentence. It seems clear enough without it.

Reply:
Same for the time description.

Reply:
It's not clear to me whether the rounding happens when you use a suffix unit for this or time either.

Reply:
The rounding does not happen -- that's what I tried to indicate with `as-is`, but looks like I didn't do a good job of that. Will reword.